jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_POST

```diff
@@ -42,6 +41,6 @@ status=$?
 # Remove the Temporary working directory
 ##########################################
 cd "${DATAROOT}" || exit 1
-[[ "${KEEPDATA}" = "NO" ]] && rm -rf "${DATA}"
```

Comment:
Can you remove this line if it is no longer active?

Reply:
Still used @WalterKolczynski-NOAA , just forgot to revert.